replay_build_scripts/lint.mjs

```diff
@@ -1,7 +1,15 @@
 import * as fs from 'fs';
 import * as eslint from 'eslint';
 
-let replayText = fs.readFileSync(process.argv[2], 'utf8');
+import * as path from 'path';
+import { fileURLToPath } from 'url';
+
+const __dirname = path.dirname(fileURLToPath(import.meta.url));
+
+let lintFile = process.argv[2] ||
+    path.join(__dirname, "..", "third_party", "blink", "renderer", "bindings", "core", "v8", "record_replay_interface.cc");
```

Comment:
FYI: You can just use `a/b/c` instead of `path.join` for constant parts of your path. It works on all systems, even Windows :)

Reply:
heh thanks for that - I always feel kinda weirded out doing that.  will change it back :)